### PR TITLE
Add flow standard

### PR DESCRIPTION
**Pull request by tychota**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/contributing/standard.s.md">standard.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/contributing/standard.s.md">standard.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/flowtype/flowtype.s.md">flowtype.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/flowtype/flowtype.s.md">flowtype.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/flowtype/flowtype.s.md">flowtype.s.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/flow-standard/flowtype/flowtype.s.md">flowtype.s.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: flowtype/flowtype.s.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# [Standard] Flow Type
+
+## Owner: [Tycho Tatitscheff](https://github.com/tychota)
+
+## Why
+
+Flowtype is awesome.:
+- it successfully replace unit tests for some tasks, like ensuring that the differents parts of the app, remains in sync with each others.
+- it give you autocompletes
```
Comment (4ian) [inline flowtype/flowtype.s.md]: *autocompletion

File: flowtype/flowtype.s.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# [Standard] Flow Type
+
+## Owner: [Tycho Tatitscheff](https://github.com/tychota)
+
+## Why
+
+Flowtype is awesome.:
+- it successfully replace unit tests for some tasks, like ensuring that the differents parts of the app, remains in sync with each others.
```
Comment (4ian) [inline flowtype/flowtype.s.md]: replaces

File: flowtype/flowtype.s.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# [Standard] Flow Type
+
+## Owner: [Tycho Tatitscheff](https://github.com/tychota)
+
+## Why
+
+Flowtype is awesome.:
```
Comment (4ian) [inline flowtype/flowtype.s.md]: awesome:

**Comment by 4ian** [comment]
Les pratiques préconisées me semblent bien !

File: dangerfile.js [hunk 4]
```diff
@@ -36,8 +36,8 @@ for (let standardFile of standardFiles) {
   if (!fileContent.match(/## Owner: .+/)) fail(`**${fileUrl}**: Standard doesn't have an *Owner*, could it be you?`);
   if (!fileContent.match(/## Checks/))
     fail(`**${fileUrl}**: Standard doesn't have a *Checks* part, how could you call this an Standard?`);
-  if (!fileContent.match(/## Bad Examples/)) fail(`**${fileUrl}**: You failed to mention a *Bad Examples*`);
-  if (!fileContent.match(/## Good Examples/)) fail(`**${fileUrl}**: You failed to mention a *Good Examples* `);
+  if (!fileContent.match(/Bad Examples?/i)) fail(`**${fileUrl}**: You failed to mention a *Bad Examples*`);
```
Comment (yleflour) [inline dangerfile.js:39]: passe le en warn, trop relou sinon

File: dangerfile.js [hunk 5]
```diff
@@ -36,8 +36,8 @@ for (let standardFile of standardFiles) {
...
+  if (!fileContent.match(/Good Examples?/i)) fail(`**${fileUrl}**: You failed to mention a *Good Examples* `);
```
Comment (yleflour) [inline dangerfile.js:40]: passe le en warn, trop relou sinon

File: flowtype/flowtype.s.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+class Comp from Reeeeeeeact.Component {
+
+}
+```
+
+If you make a typo in `React`, you can don't see it.
+
+### Exact object types
+
+#### ✅ **Good example**
```
Comment (yleflour) [inline flowtype/flowtype.s.md:56]: Start with bad then good

**Review by yleflour — CHANGES_REQUESTED** [review]

**Pull request merged**